src/assets/scss/style.scss

```diff
@@ -1,202 +0,0 @@
-$main-color: #305f8a;
-@mixin transition {
-  transition: all .2s ease-in-out; 
-
-  &:hover {
-    transform: scale(1.1);
-    font-weight: 700;
-  };
-}
-@mixin borderbotom {
-  &::before {
-    content: "";
-    position: absolute;
-    display: block;
-    width: 120px;
-    height: 1px;
-    background: #212529;
-    bottom: 0px;
-    left: calc(50% - 60px);
-  }
-}
-
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
-}
-
-
-.btn-primary {
-  background-color: $main-color;
-}
-.btn-outline-secondary {
-  color: #FFF;
-  border-color: $main-color;
-}
-a {
```

Comment:
Would be nice to have more structure in this file. Like constants, functions then elements then classes etc. Also some formatting like blank line between definitions would be helpful.